test_ped.py

```diff
@@ -8,7 +8,16 @@
 
 import ped
 from ped.guess_module import guess_module
-
+import importlib
+import os
+
+def test_dir_opening():
+    ped_obj = importlib.import_module('ped')
```

Comment:
Why is this necessary? Can you just pass `ped`?

Reply:
Not necessary. Thanks.